apps/builder/app/builder/shared/commands.ts

```diff
@@ -504,6 +504,18 @@ export const { emitCommand, subscribeCommands } = createCommandsEmitter({
       name: "wrapInLink",
       handler: () => wrapIn("Link"),
     },
+    {
+      name: "wrapInButton",
+      handler: () => wrapIn("Button"),
+    },
+    {
+      name: "wrapInHFlex",
+      handler: () => wrapIn("HFlex"),
+    },
+    {
+      name: "wrapInVFlex",
+      handler: () => wrapIn("VFlex"),
+    },
```

Comment:
We don't have these components to wrap with. It's up to designer to wrap with box and give it display flex and direction.